app/config/config_prod.yml

```diff
@@ -1,5 +1,6 @@
 imports:
   - resource: config.yml
```

Comment:
at the moment it is, since there is no particular difficulty to set it up (part of the formula), while it removes New Relic because it needs an account to be registered